pyproject.toml

```diff
@@ -18,3 +18,10 @@ showcontent = true
 directory = "bugfix"
 name = "Bug fixes:"
 showcontent = true
+
+[tool.isort]
```

Comment:
that's wrong, see [plone/meta configuration](https://github.com/plone/meta/blob/master/config/default/pyproject.toml.j2#L37-L38)

Reply:
well, essentially the same, but good to know.